config/secret.exs.example

```diff
@@ -0,0 +1,41 @@
+import Config
+
+config :contact_form,
+  deliver_emails_to: "example@email.com"
+
+config :contact_form, ContactForm.Mailer,
+  adapter: Bamboo.SMTPAdapter,
+  server: "smtp.domain",
+  hostname: "your.domain",
+  port: 1025,
+  username: "username",
+  password: "password",
+  tls: :if_available,
+  allowed_tls_versions: [:tlsv1, :"tlsv1.1", :"tlsv1.2"],
+  tls_log_level: :error,
+  tls_verify: :verify_peer,
+  tls_cacertfile: "/somewhere/on/disk",
+  tls_cacerts: "",
+  tls_depth: 3,
+  tls_verify_fun: {&:ssl_verify_hostname.verify_fun/3, check_hostname: "example.com"},
+  ssl: false,
+  retries: 1,
+  no_mx_lookups: false,
+  auth: :if_available
+
+signing_salt = raise "Please set signing_salt"
```

Comment:
WOW! THIS IS GENIUS!

Reply:
I think that the same thing should go into "Mailer" config, right? Values that don't have defaults should be tagged with "raise".